README.rst

```diff
@@ -59,7 +59,7 @@ It is implemented as a set of `Mako templates <http://makotemplates.org>`_ and p
 
 **nimi-python** supports all the Operating Systems supported by the underlying driver.
 
-**nimi-python** supports Python 2.7, 3.4 and later using CPython or PyPy.
+**nimi-python** follows Python Organization's life cycle plan. Current versions supported: Python 3.5 and later using CPython or PyPy.
```

Comment:
"Python Software Foundation" is the correct name. And link to doc.

Reply:
Blurb now states:
**nimi-python** follows `Python Software Foundation <https://devguide.python.org/#status-of-python-branches>`_ support policy for different versions. At this time this includes Python 3.5 and above using CPython or PyPy.